cmd/vela-worker/exec.go

```diff
@@ -149,7 +162,7 @@ func (w *Worker) exec(index int, config *library.Worker) error {
 		Hostname:            w.Config.API.Address.Hostname(),
 		Runtime:             w.Runtime,
 		Build:               item.Build,
-		Pipeline:            item.Pipeline.Sanitize(w.Config.Runtime.Driver),
+		Pipeline:            &execBuildExecutablePipeline,
```

Comment:
```suggestion
		Pipeline:            pipeline.Sanitize(w.Config.Runtime.Driver),
```
we still want that sanitize function there 👍

Reply:
I updated the code to reflect that